### DISCUSSION TITLE
I published a packages support gRPC client with interceptors and React

### DISCUSSION THREAD

**Discussion by nixjs** [grpc/grpc-web#1255] [Show and tell]
# gRPC Client Toolkit is the port of [gRPC-Web](https://grpc-client-toolkit.vercel.app/docs/intro#grpc-web) library.

## Features
- Typescript first-class support.
- Set default gRPC configs.
- Supports connect the gRPC Service Promise and Callbacks.
- Get all the methods in gRPC Service.
- Interceptors.
- Easy to install, update and support thanks to npm packages.

## Packages

This library is organized into small packages with few dependencies.

| Package                                          | Description                                                                           | npm                                                                                           |
| ------------------------------------------------ | ------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
| [GRPC Client Toolkit](https://grpc-client-toolkit.vercel.app/docs/grpc-core/installing) | This library is intended for both JavaScript and TypeScript usage from a web browser. | [`@nixjs23n6/core`](https://github.com/nixjs/grpc-client-toolkit/tree/master/packages/core)   |
| [GRPC Client React](https://grpc-client-toolkit.vercel.app/docs/grpc-react/installing)  | A react context which helps you to deal with gRPC web.                                | [`@nixjs23n6/react`](https://github.com/nixjs/grpc-client-toolkit/tree/master/packages/react) |

**Comment by exwyezed** [comment]
@nixjs Hi mate, what's the current state of these packages? looks like you took down them from NPM and also the GH repo seems it's gone. It looked promising and after reading the docs it's exactly what I'm looking to use for my projects.

If you decided to take it down for xyz reason (maybe you found an existing alternative), please share your thoughts as I would be really interesting in hearing them.

Cheers.

**Reply by nixjs** [reply]
Please check again. If you need any help, feel free to contact me. @alfonmga

**Reply by nixjs** [reply]
https://grpc-client-toolkit.vercel.app/docs/grpc-core/api
